### ISSUE TITLE
python virtual env activation script isn't triggered on ssh remote projects

### ISSUE THREAD

**Issue by lcdlyxrqy** [microsoft/vscode-remote-release#6140]
- VSCode Version: 1.63.2
- Local OS Version: Windows 11 Pro 21H2 22000.376
- Remote OS Version: Ubuntu 16.04.6 LTS
- Remote Extension/Connection Type: SSH
- Logs: None

Steps to Reproduce: 
(with python extension)
1. Close the remote python project and make sure this is the last vscode instance (then when you reopen vscode this project is opened automatically)
2. Reopen vscode. When you navigate to the terminal view, you will find the terminal instance doesn't trigger python virtual env activation script automatically. (newly opened terminal instances do)

Does this issue occur when you try this locally?: No
Does this issue occur when you try this locally and all extensions are disabled?: This issue involves python extension so I guess I can only answer no

**Comment by tanhakabir** [comment]
I don't believe this is a feature outside of Remote SSH either? When I close VS Code with only one workspace that had an activated environment. And then I reopen VS Code, I don't see that environment activated again.

I'm going to transfer this to the Python repo for other feedback!
